ci/cloudbuild/builds/check-abi.sh

```diff
@@ -85,6 +85,9 @@ for lib in "${libraries[@]}"; do
     io::log "Report file: ${report}"
     w3m -dump "${report}"
   fi
+  # Replaces the (old) expected dump file with the (new) actual one.
```

Comment:
Consider doing this *only* if there is a difference, i.e., move this inside the `if ! abi-compliance-checker` block.

Reply:
I considered this, but we update these .gz files each month with our releases, and in that case, we need the gz files to be updated even when there's no compliance failure.